### ISSUE TITLE
logo messed up for 0x66eb65D7Ab8e9567ba0fa6E37c305956c5341574

### ISSUE THREAD

**Issue by ligi** [MyEtherWallet/ethereum-lists#865]
looks like this:

```json
 "logo": {
      "28": "",
      "https://helex.world/logoescan.png": "",
      "ipfs_hash": ""
    },

```

btw.: any of you coming to DevCon4 or events around it for a chat and perhaps remove some friction? cc @kvhnuke

**Comment by kvhnuke** [comment]
fixed in 08fc1c851b72f295aeb51a7ae6c1a73456fa4d2a
yea we will be there :) would love to meet!

**Comment by ligi** [comment]
great! looking forward to it
